src/public/style/main.scss

```diff
@@ -1,6 +1,7 @@
 @import 'env-vars';
 @import 'globals';
 @import 'gironde-breadcrumb';
+@import '../../shared/style/stackchart';
```

Comment:
Question à 2 sous : pourquoi le charger "globalement" dans `main.scss` au lieu de garder la dépendance "locale" et propre à chaque module la nécessitant ?

Reply:
Parce que ça le charge plusieurs fois :-(
Honnêtement, je suis dégouté de ce comportement du `@import` en Sass.
`@import` en Sass, c'est un copier/coller (import in situ), pas un import de module comme on l'a en JS (sans duplicata)

J'ai fait beaucoup l'erreur de charger plusieurs fois la même chose parce que je ne savais pas. Ca n'a pas de conséquence à part le poids du CSS. 
Mais quand je rencontre un cas qui me gratte, je le corrige sur le moment (comme ici). (ça me gratte quand j'essaye de désactiver une règle CSS en live et qu'un de ses duplicata prend le relais :-p)

Reply:
Oh tristesse, j'étais persuadé du contraire, que ça ne dupliquait pas :-(